packages/features/bookings/Booker/Booker.tsx

```diff
@@ -165,12 +170,8 @@ const BookerComponent = ({
 
   const shouldShowFormInDialog = shouldShowFormInDialogMap[layout];
 
-  if (bookerState === "loading") {
-    return null;
-  }
-
   return (
-    <>
+    <div className={bookerState === "loading" ? "invisible" : "visible"}>
```

Comment:
Instead of not rendering the child components, let them render invisible so that they can fetch their own data already.